src/components/Filters/schemas/BooleanFilter.tsx

```diff
@@ -0,0 +1,22 @@
+import type {FilterSchema} from './types';
+
+export function createBooleanFilter(
+    id: string,
+    name: string,
+    options?: {
+        emptyValueText?: string;
+    },
+): FilterSchema<true> {
```

Comment:
It is looks strange a little bit. Why not `boolean`?

Reply:
That's how boolean filter works, if it's applied - value will be true, otherwise there will be no value on `filtersValues` object at all